src/evergreen_names.rs

```diff
@@ -29,6 +29,9 @@ pub const ADD_GIT_TAG: &str = "add git tag";
 // Functions for generating tasks.
 pub const GENERATE_RESMOKE_TASKS: &str = "generate resmoke tasks";
 
+// Functions for running tests.
+pub const RUN_RESMOKE_TASK: &str = "run tests";
```

Comment:
Seems confusing. Maybe rename it to `RUN_RESMOKE_TESTS` or just `RUN_TESTS`?

Reply:
Changed to `RUN_TEST_TESTS`. Adding some comments for posterity.

I'm a bit ambivalent about the naming. Conceptually, I think this function is both parallel to `_gen` tasks and downstream from them by being the generated task.

For the purposes of task generation logic in this codebase, including burn_in, we treat existing instances of this function as a part of the input for task generation, which means it's treated as an equal to the `_gen` task definitions. I therefore named it similar to `generate_resmoke_tasks` to avoid potential confusion of handling "generate_resmoke_tasks" and "run_tests" the same way.